### ISSUE TITLE
Escaped HTML for the Author Name on purpose?

### ISSUE THREAD

**Issue by asuh** [pfefferle/ZenPress#6]
On line 371, you have `esc_html( get_the_author() )`, which is producing a span with attributes, the authors name and the element brackets as entities. So instead of the author's name showing up, it shows up the span as the name.

I think you want this:
`<span itemprop="name"><span class='p-author h-card'>author</span>`

What you're getting is this:
`<span itemprop="name">&lt;span class=&#039;p-author h-card&#039;&gt;author&lt;/span&gt;</span>`

**Comment by pfefferle** [comment]
Are you using the uf2 plugin, if so, please reinstall it, ZenPress is fully microformats2 compatible.

**Comment by asuh** [comment]
Yes, I am using Microformats 2 plugin with the theme. As instructed, I deactivated, deleted it and then reinstalled it. The result is the same.

**Comment by pfefferle** [comment]
Have you reinstalled/deactivated the uf2 plugin?

**Comment by asuh** [comment]
Correct, that's what I did after your first suggestion.

**Comment by pfefferle** [comment]
What I meant was: you can deinstall it completely, because it is only a polyfill for Themes that do not support Microformats. ZenPress works perfect without the plugin, because it implements Microformats natively.

**Comment by asuh** [comment]
Oh! That fixed it. But it raises an issue with other themes that might have Microformats already in use, which is currently very few. I did not see anything on the README of this theme stating which default IndieWeb plugins not to use like uf2. 

Anyway, thank you!

**Comment by pfefferle** [comment]
You are right and we are discussing this a lot in the indieweb plugin and uf2 plugin issues.

**Comment by pfefferle** [comment]
We are discussing ways to autodisable/autoenable the plugin depending on the theme

**Comment by pfefferle** [comment]
I had a deeper look at the code and we already implemented a mechanism that I forgot to implement into ZenPress.

I only added it to my former theme SemPress https://github.com/pfefferle/SemPress/blob/master/sempress/functions.php#L102

uf2 Plugin: https://github.com/indieweb/wordpress-uf2/blob/master/wp-uf2.php#L25

I will add the line soon, so you will not have this problem any more, even if you re-install the plugin.

Thanks for filing the issue.
